### DISCUSSION TITLE
为什么新版本的docker已经自带docker compose，还要重新安装一个docker-compose

### DISCUSSION THREAD

**Discussion by seepine** [1Panel-dev/1Panel#6356] [General]
<img width="360" alt="image" src="https://github.com/user-attachments/assets/82be616d-a14e-4251-9e00-d039f7284c24">

**Comment by wanghe-fit2cloud** [comment]
为了兼容部分 1Panel 服务器上安装的低版本 Docker 服务，1Panel v2 版本将改用 `docker compose` 方式来管理和编排应用。

**Reply by seepine** [reply]
我意思是，这可以通过docker版本检测自动判断

**Reply by seepine** [reply]
这应该可以算一个优化的点，提出该问题是因为刚才安装的时候卡在这边挺久，尽管大小才50MB不知道是不是从外网下载导致，细看发现是在安装docker-compose，如果自动判断那就可以省略这个步骤了，现在安装的新用户docker基本都是新版
